apps/demos/Demos/Chat/Overview/Angular/app/app.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="demo-container">
+    <dx-chat [user]="currentUser" [items]="messages$ | async" [typingUsers]="userChatTypingUsers$ | async"
+        (onMessageEntered)="onMessageEntered($event)" (onTypingStart)="userChatOnTypingStart()"
+        (onTypingEnd)="userChatOnTypingEnd()"></dx-chat>
+    <dx-chat [user]="supportAgent" [items]="messages$ | async" [typingUsers]="supportChatTypingUsers$ | async"
+        (onMessageEntered)="onMessageEntered($event)" (onTypingStart)="supportChatOnTypingStart()"
```

Comment:
Use indent with two spaces for this file.

Reply:
7d3321bd01507196946a99f35c01832ec2021efd